<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">fingeRNAt visualization plugin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>logo_fingernat160.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>418</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;fingeRNAt visualization plugin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>411</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Process and visualize interaction data calculated with fingeRNAt.py with the -detail switch.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>441</width>
         <height>204</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QSpinBox" name="spinBox_transparency">
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="button_browse">
          <property name="text">
           <string notr="true">Browse...</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>remove all fingeRNAt objects</string>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="input_filename">
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="button_sample">
          <property name="toolTip">
           <string>Load sample data and proceed</string>
          </property>
          <property name="text">
           <string>Load sample data</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string notr="true">Filename</string>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>create receptor preferences objects</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="button_remove">
          <property name="font">
           <font>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="toolTip">
           <string>Remove all objects created by this plugin</string>
          </property>
          <property name="text">
           <string>Remove now!</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="newObjectsPrefix">
          <property name="toolTip">
           <string extracomment="All new objects will have this prefix in their names"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QCheckBox" name="checkBox_LigandPreferences">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>use this prefix for new objects</string>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>create pocket preferences objects</string>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="checkBox_ReceptorPreferences">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>transparency of objects</string>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>441</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="button_proceed">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string notr="true">Proceed!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_close">
          <property name="text">
           <string notr="true">Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <zorder>layoutWidget</zorder>
      <zorder>label</zorder>
      <zorder>label_2</zorder>
      <zorder>layoutWidget</zorder>
      <zorder>label_3</zorder>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>391</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The fingeRNAt visualization plugin, created by Filip Stefaniak (fstefaniak@iimcb.gov.pl) and Natalia Szulc; 2021.&lt;/p&gt;&lt;p&gt;Laboratory of Bioinformatics and Protein Engineering&lt;br/&gt;International Institute of Molecular and Cell Biology in Warsaw&lt;br/&gt;Ks. Trojdena 4, 02-109 ​Warsaw, Poland.&lt;/p&gt;&lt;p&gt;This plugin works with data generated by the &lt;a href=&quot;https://github.com/n-szulc/fingeRNAt/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;fingeRNAt&lt;/span&gt;&lt;/a&gt; program.&lt;/p&gt;&lt;p&gt;The source code and the tutorial is available at:&lt;br/&gt;&lt;a href=&quot;https://github.com/n-szulc/fingeRNAt/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;https://github.com/n-szulc/fingeRNAt/&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;

&lt;p&gt;The source code of this pymol plugin:&lt;br/&gt;&lt;a href=&quot;https://github.com/filipsPL/fingernat-pymol-plugin&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;https://github.com/filipsPL/fingernat-pymol-plugin&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;


&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>150</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>github32.png</pixmap>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
